gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/ResourceManagerException.java

```diff
@@ -29,11 +32,32 @@
  */
 public class ResourceManagerException extends BaseServiceException {
 
-  private static final long serialVersionUID = 6841689911565501705L;
-  private static final int UNKNOWN_CODE = -1;
+  // see https://cloud.google.com/resource-manager/v1/errors/core_errors
+  private static final Set<Error> RETRYABLE_ERRORS = ImmutableSet.of(
+      new Error(503, null),
+      new Error(500, null),
+      new Error(429, null),
+      new Error(403, "concurrentLimitExceeded"),
```

Comment:
Nit, fine as is but could be optimized if reason was a `Set<String>`.
Also, Error could have a package scope `isRetryable` for doing the internal check of code and reason matching.